core/iwasm/common/wasm_runtime_common.c

```diff
@@ -3929,11 +3937,108 @@ wasm_runtime_get_export_type(WASMModuleCommon *const module, int32 export_index,
         const WASMExport *wasm_export = &wasm_module->exports[export_index];
         export_type->name = wasm_export->name;
         export_type->kind = wasm_export->kind;
+        if (wasm_export->kind == EXPORT_KIND_FUNC) {
+            export_type->u.func_type =
+                wasm_module
+                    ->functions[wasm_export->index
+                                - wasm_module->import_function_count]
+                    ->func_type;
+        }
         return;
     }
 #endif
 }
 
+int32
```

Comment:
Prefer not to change it from uint32 to int32, the API can require that the developer must input a valid func_type, like some libc APIs. It is the responsibility of the developer to pass a valid func_type to the API.